integration_tests/mockApis/prisonRegisterApi.ts

```diff
@@ -11,9 +11,7 @@ const stubGetAllPrisons = (): SuperAgentRequest =>
     response: {
       status: 200,
       headers: { 'Content-Type': 'application/json;charset=UTF-8' },
-      jsonBody: {
-        content: allPrisons,
-      },
+      jsonBody: allPrisons,
```

Comment:
Minor correct to this stub. The timeline tests above are the first to use this stub which highlighted the problem